BuildMsys2.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+SOURCE_DIR=$PWD
+OUTPUT_DIR="build_msys2"
+
+# Ensure we are inside the repository folder
+if [ ! -f "CMakeLists.txt" ]; then
+    echo "error: file not found: CMakeLists.txt"
+    exit 1
+fi
+
+# Make output build folder
+if [ "$#" -eq 1 ]; then
+    OUTPUT_DIR=$1
+else
+    if [ ! "$#" -eq 0 ]; then
+        echo "error: too many arguemnts"
+        echo "usage: BuildLinux.sh [OUTPUT_DIR]"
```

Comment:
`BuildLinux.sh` should probably be renamed to `BuildMsys2.sh` here.

Reply:
Thank you.